### DISCUSSION TITLE
part-ii/2021-68/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#344] [Question comments]
Part II 2021 Galois Theory: Paper 1, Section II, 18I 

https://questions.tripos.org/part-ii/2021-68/

**Comment by artie2000** [comment]
<details>
<summary>Find a subfield of $\mathbb{C}$ which is a splitting field...</summary>

$L=\mathbb{Q}(\omega,\sqrt[3]{2})$.

Unique since $\mathbb{Q}$-isomorphisms take distinct roots of $X^3-2$ to distinct roots of $X^3-2$ (in $\mathbb{C}$), and all of these roots are already in $L$.

</details>

The last part is an example in 2021 lectures. For $\text{Aut}(M/\mathbb{Q})$, restrict automorphisms of $\text{Aut}(L/\mathbb{Q})$.
